xAPI-Data.md

```diff
@@ -167,8 +167,9 @@ Statements would mean that the Statement would not be interoperable with all LRS
 
 * <a name="2.2.s4.b1"></a>The LRS MUST reject Statements
     * <a name="2.2.s4.b1.b1"></a>with any null values (except inside extensions).
-    * <a name="2.2.s4.b1.b2"></a>with strings where numbers are required, even if those strings contain numbers.
-    * <a name="2.2.s4.b1.b3"></a>with strings where booleans are required, even if those strings contain booleans.
+    * <a name="2.2.s4.b1.b2"></a><a name="2.2.s4.b1.b3"></a>where the wrong data type is used for example, with strings 
```

Comment:
I still think you need a comma before "for example". Otherwise, it's way too run-on, especially for a clause with two examples.